### ISSUE TITLE
Admob not showing ad with real id

### ISSUE THREAD

**Issue by erperejildo** [flutter/flutter#51404]
I get no errors trying with the test one, just with my Admob IDs:
`PlatformException(load_failed_ad, cannot reload a failed ad, id=862739698, null)`

Banner and interstitial are failing on emulator and on real device (Xiaomi Redmi Note 8T). Just tried on Android.

All my IDs and setup is correct.

Doctor:
```
[√] Android toolchain - develop for Android devices (Android SDK    version 28.0.3)
    • Android SDK at D:\Programas\Android\sdk
    • Android NDK location not configured (optional; useful for 
      native profiling support)
    • Platform android-29, build-tools 28.0.3
    • ANDROID_HOME = D:\Programas\Android\sdk
    • Java binary at: D:\Programas\Android\Android
      Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b03)
    • All Android licenses accepted.

[√] Android Studio (version 3.5)
    • Android Studio at D:\Programas\Android\Android Studio     
    • Flutter plugin version 43.0.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b03)

[√] VS Code (version 1.42.1)
    • VS Code at C:\Users\d15_1\AppData\Local\Programs\Microsoft      VS Code
    • Flutter extension version 3.8.1

[√] Connected device (1 available)
    • AOSP on IA Emulator • emulator-5554 • android-x86 •       
      Android 9 (API 28) (emulator)
```

**Comment by iapicca** [comment]
Hi @erperejildo 
if you are using [firebase_admob](https://pub.dev/packages/firebase_admob)
please open the issue in the dedicated [github](https://github.com/FirebaseExtended/flutterfire/issues).

Otherwise
can you please provide your full `flutter doctor -v` ,
your `flutter run --verbose` 
and your `pubspec.yaml` 
or if possible a reproducible minimal code sample.
Thank you

**Comment by erperejildo** [comment]
I was creating the issue there and when I ran `flutter run --verbose` (I didn't run this project for a couple of days) I saw no errors this time.

So I tested the banner and the interstitial and both are working fine.

I've read on some places that when you create the app in Admob it can take until 1 week to have these ads ready. For my previous app this was straight away and with this one I wasn't getting a proper error log, just `load_failed_ad`.

So I guess this is resolved now.

If I see anything weird from now on I'll create the issue on the other side.
